projects/ngrx.io/content/guide/signals/signal-store/index.md

```diff
@@ -49,6 +49,27 @@ The `BookSearchStore` instance will contain the following properties:
 
 <div class="alert is-helpful">
 
+Similar to `signalState`, `withState` accepts user-defined `WritableSignal` 
+instances as part of the state.
+
+```ts
+import { signalStore, withState } from '@ngrx/signals';
+import { Book } from './book';
+
+const booksSignal = signal<Book[]>([]);
+
+export const BookSearchStore = signalStore(
+  withState({
+    books: booksSignal,
+    isLoading: false
+  })
+);
```

Comment:
and an example that shows how it works (note: signal suffix is removed from the variable name):

```suggestion
import { signalStore, withState } from '@ngrx/signals';
import { Book } from './book';

const books = signal<Book[]>([]);

const BookSearchStore = signalStore(
  withState({
    // 👇 Providing an external signal as part of the initial state.
    books,
    isLoading: false,
  })
);
```